afterburners/client.lua

```diff
@@ -26,52 +23,46 @@ else
 end
 	
 
-Citizen.CreateThread(function()
```

Comment:
why?

Reply:
This is the exact same as CreateThread, as in 
`
Citizen.CreateThread = CreateThread
`
So in theory is faster

Reply:
No? Why would it be faster? This is purely cosmetic, and it's not in the scope of a PR to change the general coding style, same goes for any other Citizen.* call

Reply:
Its literally the same thing and just makes the code look neater?